ironfish/src/rpc/server.ts

```diff
@@ -73,4 +76,21 @@ export class RpcServer {
       this._startPromise = promise
     }
   }
+
+  /** Authenticate the RPC request */
+  authenticate(requestAuthToken: string | undefined): boolean {
+    if (requestAuthToken) {
+      this.logger.debug(`Missing Auth token in RPC request.`)
```

Comment:
we don't want this because it can let an attacker flood our logs

Reply:
I want to keep the debug log for now in case I need to debug. I will clean it once the feature is launched successfully